docs/docs/transactions.md

```diff
@@ -84,11 +91,13 @@ sequelize.transaction(function (t1) {
 });
 ```
 
+### Nested/Partial transactions
```

Comment:
The code doesn't actually show a nested transaction though.
Not sure this headline fits with the CLS stuff at all actually